src/data/sql/brp/create.nationaliteiten.sql

```diff
@@ -0,0 +1,94 @@
+CREATE TABLE brp_prep.nationaliteiten AS
+
+  SELECT
+    nat."BSN"::varchar                                                               AS burgerservicenummer,
+    nat."Anummer"::varchar                                                           AS anummer,
+    JSONB_BUILD_OBJECT( -- nationaliteit
+      'code', nat."NationaliteitCode"::varchar,
+      'omschrijving', nat."NationaliteitOms"::text
+      )                                                                              AS nationaliteit,
+    JSONB_BUILD_OBJECT( -- nationaliteit verkrijging
+      'code', nat."VerkrijgingCode"::varchar,
+      'omschrijving', nat."VerkrijgingOms"::text
```

Comment:
varchar/text ?

Reply:
Ik gebruik `text` ipv `varchar` waar ik niet weet hoe lang zou de content van de kolom zijn (bv, omschrijving of reden). We werken nu met een test dat en ik heb geen idee hoe lang zou de echte data zijn. Dus ik geef beter text dan een beperkte aantal char's

Reply:
Ik gebruik (voor zekerheid) "text" ipv "varchar" in de kolommen waarvan ik niet zeker weet hoe lang de content kan zijn, bv (reden, omschrijving).
Geen bijzonderheden qua lengte van de inhoud van de kolommen met deze test data, maar ik weet niet hoe lang zou de echte data zijn!

Reply:
varchar zonder specifier of text gedragen zich hetzelfde volgens mij
https://www.postgresql.org/docs/current/datatype-character.html

Reply:
ik zal het later gelijktrekken. in de nieuwe sql files wordt varchar gebruikt.